logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java

```diff
@@ -13,84 +13,85 @@
 
 package tech.pegasys.teku.logging;
 
-import org.apache.logging.log4j.Level;
+import static tech.pegasys.teku.logging.ColorConsolePrinter.print;
+
+import com.google.common.primitives.UnsignedLong;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.logging.ColorConsolePrinter.Color;
 
 public class StatusLogger {
 
   public static final StatusLogger STATUS_LOG = new StatusLogger("stdout");
 
-  public enum Color {
-    RED,
-    BLUE,
-    PURPLE,
-    WHITE,
-    GREEN
-  }
+  private final Logger logger;
+  private boolean enabled;
 
-  private static final String resetCode = "\u001B[0m";
+  private StatusLogger(final String name) {
+    this.logger = LogManager.getLogger(name);
+  }
 
-  private final Logger logger;
+  public void setEnabled(final boolean enabled) {
+    this.enabled = enabled;
+  }
```

Comment:
I'm not a huge fan of making this singleton mutable.  Why is this needed?
If it is needed, it would need to be thread safe which it isn't currently.